### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by ivy-lli**

### CODE REVIEW

File: scripts/download-engine.cts
Comment (ivy-lli) [inline scripts/download-engine.cts:1]: it is normally not a good idea to mix esm (import) and commonjs (require) styles
And when a file uses commonjs require imports then best add a c to the file extension

File: scripts/download-openapi.ts
Comment (ivy-lli) [inline scripts/download-openapi.ts:1]: same here, but as all stuff is available via esm import i used here esm imports

File: package.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
   "scripts": {
     "build": "lerna run build",
     "build:production": "lerna run build:production",
-    "download:engine": "ts-node scripts/download-engine.ts",
-    "download:openapi": "ts-node scripts/download-openapi.ts",
+    "download:engine": "node --experimental-strip-types scripts/download-engine.cts",
+    "download:openapi": "node --experimental-strip-types scripts/download-openapi.ts",
```
Comment (ivy-lli) [inline package.json:14]: node has meanwhile builtin type striping, even if experimental for this it should to it's job :)

File: extension/src/engine/api/jsonrpc.ts [hunk 2]
```diff
@@ -2,8 +2,6 @@ import { BaseRpcClient, Connection, createMessageConnection, Disposable } from '
 import { Callback, WebIdeClient } from './jsonrpc-protocol';
 import { ProcessBean } from './generated/openapi-dev';
 
-export interface WebIdeOnNotificationTypes {}
```
Comment (ivy-lli) [inline extension/src/engine/api/jsonrpc.ts:5]: fix lint problems. currently not used so I removed this stuff

File: extension/src/engine/ws-client.ts [hunk 3]
```diff
@@ -12,7 +12,9 @@ export const WebSocketClientProvider = (webSocketUrl: URL) => {
       client.animationSettings(animationSettings());
       client.onOpenEditor.set(process => handleOpenEditor(process));
       vscode.workspace.onDidChangeConfiguration(e => {
-        e.affectsConfiguration('process.animation') ? client.animationSettings(animationSettings()) : {};
+        if (e.affectsConfiguration('process.animation')) {
+          client.animationSettings(animationSettings());
+        }
```
Comment (ivy-lli) [inline extension/src/engine/ws-client.ts:17]: make linter happy

File: package.json (hunk 1 above)
Comment (ivy-lmu) [reply package.json:14]: aha cool

**Review by ivy-lmu — APPROVED** [review]
thanks :shipit:

**Pull request merged**